room_cancelation/auto_cancel_event.py

```diff
@@ -0,0 +1,57 @@
+from api.room.models import Room as RoomModel
+from api.events.models import Events as EventsModel
+from datetime import datetime
+from helpers.calendar.credentials import Credentials
+from dateutil.relativedelta import relativedelta
+from helpers.calendar.events import RoomSchedules
+
+
+class UpdateRecuringEvent():
+    def update(self):
+        query = RoomModel.query
+        now = datetime.utcnow().isoformat()+"Z"
+        next_day = (datetime.utcnow() + relativedelta(day=1)).isoformat()+"Z"
+        events = RoomSchedules().get_all_recurring_events(
+            query, now, next_day)
+        for event in events:
+            start_date = event["start_date"]
+            end_date = event["end_date"]
+            new_recurring_event = EventsModel(
+                event_id=event["recurring_event_id"],
+                room_id=event["room_id"],
+                event_title=event["event_summary"],
+                start_time=start_date,
+                end_time=end_date,
+                checked_in=False,
```

Comment:
Kindly share the reason for hard-coding `checked_in` and `cancelled`

Reply:
This is because when we are creating the event the user has not checked in or canceled it. Also the data does not come from the google API which is why we initialized them to false.

Reply:
+1 @caveinn also they are populated with future events...so in no way would it already come as checked_in